.circleci/config.yml

```
@@ -32,6 +32,11 @@ jobs:
       - image: circleci/ruby:2.4.9
     steps:
       - checkout
+      - run:
+          name: Determine if gem build is needed
+          command: |
+            git log -m -1 --name-only --pretty="format:%d" -n 1 | grep -q "version.rb" \
```

There are a couple different ways to do this -- I particularly like https://stackoverflow.com/a/39819750 -- but another form that doesn't require understanding all the ways of denoting a revision or some of `git log`s flags is `git diff`.

This will follow first parent, which should have the same outcome as above with fewer flags.
```suggestion
            git diff --name-only HEAD^..HEAD | grep -q version.rb \
```